kbase-extension/static/kbase/js/common/cellComponents/tabs/results/outputWidget.js

```diff
@@ -51,23 +54,39 @@ define([
                     thead(tr([th('Created Object Name'), th('Type'), th('Description')])),
                     tbody([
                         ...objectData.map((obj) => {
-                            const parsedType = APIUtil.parseWorkspaceType(obj.type);
-                            const objLink = a(
-                                {
-                                    class: 'kb-output-widget__object_link',
-                                    dataObjRef: obj.ref,
-                                    type: 'button',
-                                    ariaLabel: 'show viewer for ' + obj.name,
-                                    id: events.addEvent({
-                                        type: 'click',
-                                        handler: () => {
-                                            Jupyter.narrative.addViewerCell(obj.wsInfo);
-                                        },
-                                    }),
-                                },
-                                [obj.name]
-                            );
-                            return tr([td(objLink), td(parsedType.type), td(obj.description)]);
+                            const name = obj.name
+                                ? obj.name
+                                : obj.wsInfo
+                                ? obj.wsInfo[1]
+                                : 'Unknown object name';
+                            const type = obj.type
+                                ? obj.type
+                                : obj.wsInfo
+                                ? obj.wsInfo[2]
+                                : 'Missing type';
+                            const parsedType = APIUtil.parseWorkspaceType(type) || { type };
+                            const description = obj.description || 'Missing description';
+                            let objLink = '';
```

Comment:
you could initialise it to `name`, and then change it if `obj.wsInfo` is available -- saves a couple of lines.